### PR TITLE
Update gems and travis config for Ruby 2.

### PR DESCRIPTION
**Pull request by mkb**

### CODE REVIEW

**Comment by halorgium** [comment]
I would prefer to continuing to use the `Gem::Version` code. 
Something like?

``` ruby
if ruby 2.0
  # ...
elsif ruby 1.9
  # ...
else
  # ...
end
```

**Comment by mkb** [comment]
Seems like indirection with no clear payoff but sure, I'll get cracking.

**Comment by halorgium** [comment]
Yer, I wrote some code previously that had some bugs, @benburkert fixed it up to use the `Gem::Version` stuff. 
We should have tests for the `Gemfile` ;)

**Comment by mkb** [comment]
Boom. Passing CI.

File: Gemfile [hunk 1]
```diff
@@ -17,9 +16,10 @@ group :test do
   gem 'rspec',    '>=2.0.0'
   gem 'realweb'
 
-  if ruby_version >= ruby_19
-    gem 'debugger' if ruby_engine == 'ruby' # MRI
-
+  if ruby_version >= ruby_20
+    gem 'em-synchrony'
+  elsif ruby_version >= ruby_19
+    gem 'debugger' if RUBY_ENGINE == 'ruby'
```
Comment (halorgium) [inline Gemfile:22]: Is this always defined?
I guess it is passing! :+1:

**Comment by halorgium** [comment]
Cheers! :heart:

**Pull request merged**